neo/Ledger/Blockchain.cs

```diff
@@ -115,9 +115,14 @@ public Blockchain(NeoSystem system, Store store)
                     }
                 }
                 if (header_index.Count == 0)
+                {
                     Persist(GenesisBlock);
+                }
                 else
+                {
                     UpdateCurrentSnapshot();
+                    MemPool.LoadPolicy(currentSnapshot);
```

Comment:
Maybe we should check this after genesis too, if we want to define in genesis the default value

Reply:
Persist `GenesisBlock`:

https://github.com/neo-project/neo/blob/9c85a069c1ddd5e9597191717cfdc661be3380b4/neo/Ledger/Blockchain.cs#L119

It calls `OnPersistCompleted()`:

https://github.com/neo-project/neo/blob/9c85a069c1ddd5e9597191717cfdc661be3380b4/neo/Ledger/Blockchain.cs#L483

https://github.com/neo-project/neo/blob/9c85a069c1ddd5e9597191717cfdc661be3380b4/neo/Ledger/Blockchain.cs#L372-L377

It has:

```c#
MemPool.UpdatePoolForBlockPersisted(block, currentSnapshot);
```

In `MemPool.UpdatePoolForBlockPersisted()`:

https://github.com/neo-project/neo/blob/9c85a069c1ddd5e9597191717cfdc661be3380b4/neo/Ledger/MemoryPool.cs#L407

So we needn't call `MemPool.LoadPolicy()` in genesis block persistence.